index.html

```diff
@@ -289,21 +289,81 @@ <h3><a class="about-connect" href="#">LET'S CONNECT</a></h3>
     </section>
 
     <!-- Contact-Form -->
-    <section class="contact-form" id="Contact">
+    <section class="contact-form" id="contact">
       <h2 class="contact-title" >Contact me</h2>
       <p class="contact-info" >If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it</p>
       
       <fieldset class="fieldset">
-        <form action="https://formspree.io/f/xaykrzrn" method="post">
+        <form action="https://formspree.io/f/xaykrzrn" method="post" id="form">
         <ul>
           <li><input type="text" class="form-input" name="name" placeholder="Your Name" maxlength="30" required></li>
-          <li><input type="email" class="form-input" name="email" placeholder="Your Email" required></li>
+          <li><input type="email" class="form-input" id="email" name="email" placeholder="Your Email" required></li>
           <li><textarea class="form-input" id="text-area" name="text-area" placeholder="Write your message here" maxlength="500" required></textarea></li>
-          <button class="contact-button" type="submit" name="button">Get In Touch</button>
+          <button class="contact-button" type="submit" id="submit" name="button">Get In Touch</button>
         </ul>
+        <span class="error" id="error'"></span>
         </form>
       </fieldset>
     </section>
 
+    <!-- MODAL SECTION -->
+    <!-- <div class="card-container" id="card-container">
+    <div class="card-container-title"><h2 id="title">Tonic</h2> <img src="./Images/Icon - Cancel.png" alt="xx" class="tonicx" id="tonicx"> </div>
+    <div><ul class="card__content--item1">
+      <li class="card__item--heading"> Canopy</li>
+      <li class="card__item--heading1"> <img src="Images/Counter.png" alt="Counter"> Back End Dev</li>
+      <li class="card__item--heading2"> <img src="Images/Counter.png" alt="Counter"> 2015</li>
+    </ul></div>
+    <div><img class="mobiletonic" id="mobiletonic" src="./Images/xweb.png" alt="MobileTonic"></div>
+    <div><p class="card__content--desc">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent</p></div>
+    <div> <ul class="card__content--item2">
+      <li class="tech__btn1">html</li>
+      <li class="tech__btn2">css</li>
+      <li class="tech__btn3">Javascript</li>
+    </ul></div>
+    <div class="bcontainer">
+      <button class="btncontainer1" id="site">See Live<img src="ivit.svg" class="btncntim"></button>
+      <button class="btncontainer" id="site">See Live</button>
+      <button class="btncontainer" id="source">See Source<img src="./Images/igit.png" class="btncntim"></button>
+    </div>
+  </div> -->
```

Comment:
- [ ] Dear @mertkantaroglu, Commented codes are against a good practice for clean code as this will make your code messy. Pleas remove commented code and remember you can always go back with the commit history to see a code you have used once.